WASM_Support.md

```diff
@@ -16,16 +16,57 @@
 
 # WASM Support
 
+oneTBB extends its capabilities by offering robust support for ``WASM``. 
+
 ``WASM`` stands for WebAssembly, a low-level binary format for executing code in web browsers. 
-It is designed to be a portable target for compilers and to be efficient to parse and execute. 
+It is designed to be a portable target for compilers and efficient to parse and execute. 
+
+By integrating oneTBB with WASM, you can take full advantage of parallelism and concurrency while working on web-based applications, interactive websites, and a variety of other WASM-compatible platforms.
```

Comment:
I don't understanding the meaning of:
'By integrating oneTBB with WASM, you can take full advantage'
Who will integrate what with what?

'oneTBB offers WASM support through the integration with'
Means that oneTBB provides integration with WASM.

Reply:
@pavelkumbrasev 
1) maybe rewrite it like that: Using oneTBB with WASM, you can......
2) should be like that: oneTBB provides integration with WASM using Emscripten* ....

Reply:
Let's apply the first one: `Using oneTBB with WASM, you can......`
And keep the second as it was originally